multimodal/src/autogluon/multimodal/data/process_image.py

```diff
@@ -102,8 +115,18 @@ def __init__(
         self.mean = None
         self.std = None
 
+        if self.prefix == MMDET_IMAGE:
+            # TODO: can we pass the config information here when we build the model?
+            assert mmcv is not None, "Please install mmcv-full by: mim install mmcv-full."
```

Comment:
Do we still need to assert `mmcv` and `mmdet` in `process_image.py `if we have asserted them in the `mmdet_image.py`?

Reply:
For now, we don't. But we can leave it here just in case. Once we figure out all the dataloader problem and complete PRs for training detectors, we can remove these asserts if not needed. Is that ok?

Reply:
Sure.